docs/validators/get-started/install-regen.md

```diff
@@ -28,18 +28,6 @@ Check out the genesis version:
 git checkout v1.0.0
 ```
 
-*For Redwood Testnet:*
-
-```bash
-git checkout v1.0.0
-```
-
-*For Hambach Testnet:*
-
-```bash
-git checkout v4.0.0
-```
-
 Build and install the `regen` binary:
 
 *For Regen Mainnet:*
```

Comment:
These are no longer needed if the instructions apply to all (i.e. mainnet and redwood)